README.md

```diff
@@ -1,16 +1,25 @@
-CLE
-===
-[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+# CLE
+[![Latest Release](https://img.shields.io/pypi/v/cle.svg)](https://pypi.python.org/pypi/cle/)
+[![Python Version](https://img.shields.io/pypi/pyversions/cle)](https://pypi.python.org/pypi/cle/)
```

Comment:
Is there a way to make it more specific than "3"? Specifically we only claim to support Python 3.8+. If not I don't think the Python version provides much use here, since Python 2 is EOL 3 years ago anyway

Reply:
I believe this is done with classifiers we define here: https://github.com/angr/cle/blob/master/setup.cfg#L13. I agree we should update them to show python 3.8-3.11 support rather than just 3.